test/utils.jl

```diff
@@ -12,6 +12,10 @@ using Test
 using LightGraphs
 using SimpleWeightedGraphs
 
+find_eps(x::Type{TN}) where TN<:Number = eps(real(TN))
```

Comment:
is it a bad idea to just extend `eps` here?

Reply:
It could work but we wouldn't benefit from it. It's just a small utility for testing and it's more predictable if it's a separate function.